admin_guide/scheduling/priority_preemption.adoc

```diff
@@ -88,51 +87,77 @@ spec:
+As the scheduler waits for the lower-priority pod to terminated, Node M becomes available. The scheduler then schedules Pod P on Node M. 
 
-Our recommended solution for this problem is to create inter-Pod affinity only towards equal or higher priority Pods.
+=== Pod priority and pod disruption budget
 
-==== Cross node preemption
+A xref:../../admin_guide/managing_pods.adoc#managing-pods-poddisruptionbudget[pod disruption budget] specifies the minimum number or percentage of replicas that must be up at a time. {product-title} supports pod dusruption budgets when preempting pods at a best effort level. The scheduler attempts to preempt pods without violating the pod dusruption budget. If no such pods are found, lower-priority pods might be removed despite their pod dusruption budget requirements.
 
-Suppose a Node N is being considered for preemption so that a pending Pod P can be scheduled on N. P might become feasible on N only if a Pod on another Node is preempted. Here’s an example:
+==== Pod priority and inter-pod affinity
+
+Pod affinity requires a new pod to be scheduled on the same node as other pods with the same label. 
+
+If a pending pod has inter-pod affinity with one or more of the lower-priority pods on a node, the schedulder cannot preempt the lower-priority pods with out viilating the affinity requirements.  In this case, the scheduler looks for another node to schedule the pending pod. However, there is no guarantee that the scheduler can find an appropriate node and pending pod might not be scheduled.
+
+To prevent this situation, carefully configure inter-pod affinity with equal-priority pods.
+
+==== Pod priority and cross-node preemption
+
+If the scheduler is considering prempting pods on a node so that a pending pod can be scheduled, the scheduler can preempts a pod on different node in order to schedule the pending pod. 
+
+For example:
 
 . Pod P is being considered for Node N.
-. Pod Q is running on another Node in the same Zone as Node N.
-. Pod P has Zone-wide anti-affinity with Pod Q (topologyKey: failure-domain.beta.kubernetes.io/zone).
-. There are no other cases of anti-affinity between Pod P and other Pods in the Zone.
-. In order to schedule Pod P on Node N, Pod Q can be preempted, but scheduler does not perform cross-node preemption. So, Pod P will be deemed unschedulable on Node N.
+. Pod Q is running on another node in the same zone as Node N.
+. Pod P has zone-wide anti-affinity with pod Q, meaning Pod P cannot be scheduled in the same zone as Pod Q.
+. There are no other cases of anti-affinity between Pod P and other pods in the zone.
+. In order to schedule Pod P on Node N, the scheduler must preempt Pod Q to remove the pod anti-affinity violation, allowing the scheduler to schedule Pod P on Node N.
+
+Pod Q can be preempted, but scheduler does not perform cross-node preemption. So, Pod P will be deemed unschedulable on Node N. 
+
+== Disabling Priority and Preemption
```

Comment:
@ravisantoshgudimetla @wjiangjay 
Do we want to include the instructions for disabling the feature? 
1. Feature gates:
`feature-gates.Priority=false` (what is the name to use?) 
2. Edit the scheduler config to set `disablePreemption=true`

Reply:
Yes, @mburke5678, that makes sense but we need to be explicit on the scheduler config file for `disablePreemption`. Users may confuse it with scheduler.json file which has predicates and priorities. It should be componentconfig file or update master-config.yaml schedulerArgs section with `disablePreemption=false`.